public/app/features/alerting/unified/utils/matchers.ts

```diff
@@ -37,6 +37,7 @@ interface MatchedInstance {
   };
 }
 
+// TODO We can probably remove this as filtering is done in the backend
```

Comment:
Should we remove it?

Reply:
I mean the whole function below this line

Reply:
Yep, let's remove if it's not referrenced